### DISCUSSION TITLE
Respawn in a different instance

### DISCUSSION THREAD

**Discussion by BomBardyGamer** [Minestom/Minestom#1943] [Help]
What's the best way to respawn someone in a new instance? My first thought is to just call setInstance and then respawn, but that seems a little bit strange if you see that you get moved to a new instance and then you seem to respawn. I think it will also double-send chunks.

I don't want a seamless transition between different instances, I want a respawn screen.

**Comment by Tofaa2** [comment]
I think themode did mention how the instance api is very weird to use and its a design flaw, your method seems to be the only decent way
